cea/plots/demand/energy_balance.py

```diff
@@ -159,25 +158,25 @@ def calc_graph(analysis_fields, data_frame):
     return graph
 
 
-
-
-
 def calc_monthly_energy_balance(data_frame, normalize_value):
     """
     calculates heat flux balance for buildings on hourly basis
 
-    :param data_frame:
+    :param data_frame: demand information of building in pd.DataFrame
+    :param normalize_value: value for normalization of thermal energy fluxes, usually GFA
     :return:
     """
 
+    # invert sign of I_rad, I_rad was previously negative but it was changed to be consistent with ISO 13790
```

Comment:
the comment "I_rad was previously negative but it was changed" is hard to understand without a historical context of the code. Is there a better way to state this?

What is I_rad like before changing signs? Why do we change signs?